plugins/plugins.go

```diff
@@ -61,9 +67,25 @@ func loadFunc(fileName, funcName string, r *http.Request) (ret string, err error
 	if err != nil {
 		return
 	}
-	// Exec (call) function name, return string
-	ret = f.(func() string)()
-	log.Println("ret plugin:", ret)
+	// Exec (call) function name, return string (In case which return status code does not matter)
+	function, ok := f.(func() string)
+
+	if !ok {
+		// It is probable that plugin function return not only json but also status code.
+		function := f.(func() (string, int))
+		retJson, code := function()
+		ret.ReturnJson = retJson
+		ret.StatusCode = code
```

Comment:
if `code` is `nil`, set default value `http.StatusOK`

when you don't set the `status code` I think it's better to have a default return of `200` (aka `http.StatusOK`)

does this proposal make sense @sod-lol @arxdsilva?

Reply:
Hi.
The type of code variable is int, so it cannot be nil unless we change the type of code variable, but don't you agree that forcing a developer to explicitly return status code minimizes the error? What is your opinion? @avelino @arxdsilva